iml-agent/src/action_plugins/stratagem/action_warning.rs

```diff
@@ -112,3 +117,63 @@ pub async fn read_mailbox(
 
     Ok(txt_path)
 }
+
+async fn fi2path(llapi: LlapiFid, fi: FidItem) -> Option<String> {
```

Comment:
what's the difference between `fi` and `fid`?

Reply:
FID is just the lustre fid, where FidItem has FID plus some per-fid data (in json form)

Reply:
I renamed it for clarity